python/cuml/solvers/qn.pyx

```diff
@@ -302,3 +398,10 @@ class QN(Base):
         del X_m
 
         return cudf.Series(preds)
+
+    def __getattr__(self, attr):
+        if self.fit_intercept:
+            if attr == 'coef_':
+                return self.coef_[0:-1]
```

Comment:
I think it would be cleaner to call `self.coef_` something else like `self.all_coefs_` or `self.coef_and_intercept_` so it's clearly different from the exposed meaning of `coef`. Could be a separate change though.

Reply:
yeah, I agree though it'd require changing tests. And also it is something that will be part of more upcoming uniformization of the code base (some classes right now use cuDF objects, others use device arrays). Will open an issue

Reply:
Opening an issue is ok by me. Definitely need to plan for a big harmonization of return values and coeffs.